fastlane/download_metadata.swift

```diff
@@ -2,11 +2,15 @@
 
 import Foundation
 
+let glotPressTitleKey = "app_store_title"
 let glotPressSubtitleKey = "app_store_subtitle"
 let glotPressWhatsNewKey = "v4.43-whats-new"
 let glotPressDescriptionKey = "app_store_desc"
 let glotPressKeywordsKey = "app_store_keywords"
-let baseFolder = "./metadata"
+
+let scriptURL = URL(fileURLWithPath: CommandLine.arguments[0], relativeTo: URL(fileURLWithPath: FileManager.default.currentDirectoryPath))
+let fastlaneDir = scriptURL.deletingLastPathComponent()
+let baseFolder = fastlaneDir.appendingPathComponent("metadata").path
```

Comment:
This is to ensure that the path for `baseFolder` is computed relative to the script's path and not to the current directory.

Made that change because I kept accidentally running `fastlane/download_metadata.swift` from the root repo during my tests… and had the script create the `metadata/` folder at the repo root instead of inside the `fastlane/` subfolder.
This is not an issue when this script is run by our release-toolkit and its `ios_update_metadata` action because [we explicitly `cd fastlane` before invoking the script in this context](https://github.com/wordpress-mobile/release-toolkit/blob/develop/lib/fastlane/plugin/wpmreleasetoolkit/helper/ios/ios_git_helper.rb#L60), but it's still sometimes useful to run the script standalone/ourselves which is why I thought this to be useful